vpr/src/base/netlist_writer.cpp

```diff
@@ -1245,7 +1245,9 @@ class NetlistWriterVisitor : public NetlistVisitor {
 
         //Add the single output connection
         {
-            auto atom_net_id = top_pb_route[sink_cluster_pin_idx].atom_net_id; //Connected net in atom netlist
+            AtomNetId atom_net_id = AtomNetId::INVALID();
+            if (top_pb_route.count(sink_cluster_pin_idx))
```

Comment:
Suggest putting a comment here saying what this is doing (as best you understand it anyway).